ACE/ace/os_include/sys/os_types.h

```diff
@@ -53,7 +57,7 @@ typedef double ACE_timer_t;
 
 #if defined (ACE_SIZEOF_LONG) && ACE_SIZEOF_LONG == 8
    typedef off_t ACE_LOFF_T;
-#elif defined (__FreeBSD__) || defined (__NetBSD__) || defined (__OpenBSD__) || defined (__APPLE__)
+#elif defined (__FreeBSD__) || defined (__NetBSD__) || defined (__OpenBSD__) || defined (__APPLE__) || defined (ACE_HAS_MUSL)
```

Comment:
In the original PR, `ACE_HAS_MUSL` was something I suggested the user was going to define, is that still the case?

Reply:
No, I forgot to take it out of this file when config-linux changed.